products/llbuildSwift/BuildSystemBindings.swift

```diff
@@ -301,8 +301,28 @@ public struct BuildKey {
     }
 }
 
+public protocol FileInfo {
```

Comment:
Can we get documentation comments on these protocols?

Reply:
Done.